malaria-app-android/src/main/java/com/peacecorps/malaria/activities/RemainderToneActivity.java

```diff
@@ -51,6 +50,12 @@ public void onClick(View v) {
                 break;
 
             case R.id.dialogButtonOKReminder:
+
+                if("".equals(path.getText()) || !isAudioFile())
```

Comment:
Use StringUtil.isEmpty function to check empty strings. Do not carry starting brace to next line.